pkg/global/config.go

```diff
@@ -0,0 +1,81 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package global
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+type ValueType interface {
+	string | int | bool | time.Duration
+}
+
+var configMap sync.Map
+
+func Init(fromConfigMaps ...*corev1.ConfigMap) {
+	configMap.Range(func(key, value interface{}) bool {
+		configMap.Delete(key)
+		return true
+	})
```

Comment:
```suggestion
	configMap.Clear()
```

Reply:
This one is still left over from previous iterations.

Reply:
There is not Clear method on sync.Map.

Reply:
https://pkg.go.dev/sync#Map.Clear